lib/build/definitions/library.js

```diff
@@ -120,15 +121,18 @@ export default function({project, taskUtil, getTask}) {
 			taskFunction: async ({workspace, dependencies, taskUtil, options}) => {
 				const generateBundleTask = await getTask("generateBundle");
 				return bundles.reduce(function(sequence, bundle) {
-					return sequence.then(function() {
+					return sequence.then(async function() {
+						const {bundleDefinition, bundleOptions} = await enhanceBundleWithDefaults(
```

Comment:
Wouldn't it be more efficient to process all bundle configs at once before running the promise sequence?

Reply:
Done

Reply:
With that, I also meant to apply the defaults for an array of bundle configurations (as it's designed in the schema), not one by one. Or is there a reason why it has to be done one by one?

Reply:
Ah, thanks!

I didn't see that initially! Now should be ok